docs/1.getting-started/7.state-management.md

```diff
@@ -33,7 +33,7 @@ Instead use `const useX = () => useState('x')`
 
 ### Basic Usage
 
-In this example, we use a component-local counter state. Any other component that uses `useState('counter')` shares the same reactive state.
+In this example, we use a component-local counter-state. Any other component that uses `useState('counter')` shares the same reactive state.
```

Comment:
It's a state assigned to a variable called _counter_, I would leave out the hyphen.
```suggestion
In this example, we use a component-local counter state. Any other component that uses `useState('counter')` shares the same reactive state.
```